mint.json

```diff
@@ -46,7 +46,13 @@
             "features/prompt-registry/snippets",
             "features/prompt-registry/release-labels",
             "features/prompt-registry/dynamic-release-labels",
-            "features/prompt-registry/function-calling",
+            {
```

Comment:
instead of this being a sub group, can you just do "Tool Calling" then "Structured Outputs"

And make sure they link to eachother in the intro paragraph